x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_model_health.tsx

```diff
@@ -207,7 +209,15 @@ export const TrainedModelHealth: React.FC<TrainedModelHealthProps> = ({
   }
   return (
     <EuiToolTip content={modelHealth.tooltipText}>
-      <EuiHealth color={modelHealth.healthColor}>{modelHealth.healthText}</EuiHealth>
+      {wrapperElement ? (
+        React.cloneElement(
```

Comment:
we are only passing children here? using cloneElement feels hacky here, can't we just do

`wrapperElem(<EuiHealth color={modelHealth.healthColor}>{modelHealth.healthText}</EuiHealth>) `or sth like that?